content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -247,21 +247,14 @@ DaemonSets and StatefulSets. CSI plugins handle a wide range of volume
 management actions in Kubernetes: provisioning/de-provisioning/resizing of
 volumes, attaching/detaching of volumes to/from a Kubernetes node and
 mounting/dismounting a volume to/from individual containers in a pod,
-backup/restore of persistent data using snapshots and cloning. CSI plugins
-typically consist of node plugins (that run on each node as a DaemonSet) and
-controller plugins.
-
-CSI node plugins (especially those associated with persistent volumes exposed
-as either block devices or over a shared file-system) need to perform various
-privileged operations like scanning of disk devices, mounting of file systems,
-etc. These operations differ for each host operating system. For Linux worker
-nodes, containerized CSI node plugins are typically deployed as privileged
-containers. For Windows worker nodes, privileged operations for containerized
-CSI node plugins is supported using
-[csi-proxy](https://github.com/kubernetes-csi/csi-proxy), a community-managed,
-stand-alone binary that needs to be pre-installed on each Windows node. Please
-refer to the deployment guide of the CSI plugin you wish to deploy for further
-details.
+backup/restore of persistent data using snapshots and cloning.
+
+CSI plugins communicate with a CSI node plugin which performs the local storage operations.
+On Windows nodes the community-managed [csi-proxy](https://github.com/kubernetes-csi/csi-proxy)
+is used.
+
+Please refer to the following [installation steps](https://github.com/kubernetes-csi/csi-proxy#installation)
```

Comment:
I would prefer to continue to have users refer to their environment/CSI plugin docs first as the environment may be already installing csi proxy for them. So how about: "Please
refer to the deployment guide of the CSI plugin you wish to deploy for further details around installation. You may also refer to the following ..."

Reply:
Updated